torch/testing/_internal/common_methods_invocations.py

```diff
@@ -500,6 +526,26 @@ def sample_inputs(self, device, dtype, requires_grad=False):
             ]
 
 
+class ShapeFuncInfo(OpInfo):
```

Comment:
Add a brief comment here explaining what this derived class is intended for (maybe something like, "Early version of a specialized OpInfo for "shape" operations like tile and roll"?)